package.json

```diff
@@ -56,6 +56,7 @@
 		"@wordpress/browserslist-config": "2.3.0",
 		"@wordpress/components": "7.0.8",
 		"@wordpress/compose": "3.0.1",
+		"@wordpress/data": "4.4.0",
```

Comment:
Why was this dependency added to the root?

```suggestion
```

Reply:
I just added it because my IDE was complaining it was not installed when checking the imports used on the `@automattic/gutenframe` package.

But you're right it's not needed on the root since it's not used on Calypso, so I removed it on 2cdc03c.